frontend/src/components/common/SyncButton.vue

```diff
@@ -85,6 +92,15 @@ const onClick = () => {
       {{ name }}
     </h3>
 
+    <span style="margin-right: 5px">Last sync date:</span>
```

Comment:
Please use classes to do this margin behavior. Might be `class="mr-1"` or something similar you'd need to check the docs. We generally try to avoid doing in-line styles as they become very hard to manage and track down for maintainability reasons.